sanity-io/structured-content-2022
# VenueNames: header & CTA, highlight main venue

# VenueNames: header & CTA, highlight main venue

## Intent

Render new fields (and logic changes) for `venuesSection` in accordance with [Figma updates](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=6355%3A18309).

## Remaining TODOs before this PR is ready for review:

- [x] ~@kmelve According to the Figma sketches, it should be possible to configure each venues to link to an arbitrary target (hence the underlined venue names in the Figma sketches). Could you please advise on the best Studio schema structure to accomplish this?~ We are scoping linking out of the first iteration covered by this PR.
- [x] @oyvind-stenhaug Styling and markup changes


## Testing this PR

1. Open https://structured-content-2022-web-git-venue-names-show-main-venue.sanity.build/
2. Scroll down below the nav block and verify that the venues section looks like indicated in the Figma sketch
